### PR TITLE
Jessiedlcs/update react native

### PR DESCRIPTION
**Pull request by jessdelacruzsantos**
## Summary
The React Native verison on the example app was 5 versions old.
## Related issues

Fix #
Updated React Native version of example app, and updated config files.
## Changelog

* message

### CODE REVIEW

File: reader-sdk-react-native-quickstart/android/app/build.gradle [hunk 1]
```diff
@@ -77,28 +109,40 @@ android {
...
+            def versionCodes = ["armeabi-v7a": 1, "x86": 2, "arm64-v8a": 3, "x86_64": 4]
             def abi = output.getFilter(OutputFile.ABI)
             if (abi != null) {  // null for the universal-debug, universal-release variants
                 output.versionCodeOverride =
                         versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
             }
+
         }
     }
-    dexOptions {
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/android/app/build.gradle]: is this not required?

File: reader-sdk-react-native-quickstart/app/screens/PermissionScreenIOS.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default class PermissionScreenIOS extends Component {
     locationbuttonHandler: null,
   }
 
-  async componentWillMount() {
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/app/screens/PermissionScreenIOS.js]: Why do we change this? get warning?

File: reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist]: We don't need a test project, can we remove it?

File: reader-sdk-react-native-quickstart/package.json [hunk 4]
```diff
@@ -1,35 +1,32 @@
 {
-  "name": "reader-sdk-react-native-quickstart",
+  "name": "RNReaderSDKSample",
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/package.json]: do we need rename?

File: reader-sdk-react-native-quickstart/android/gradle.properties [hunk 5]
```diff
@@ -31,14 +31,15 @@
...
 org.gradle.jvmargs=-Xmx4g
-org.gradle.configureondemand=true
+android.enableJetifier=true
+
 
 ## Uncomment the properties below and set their correct value.
 
 ## The Application ID displayed on the Credentials tab in the Square Application Control Panel.
-## SQUARE_READER_SDK_APPLICATION_ID=
+SQUARE_READER_SDK_APPLICATION_ID=
```
[the remark this answers is not in the document]
Comment (jessdelacruzsantos) [reply reader-sdk-react-native-quickstart/android/gradle.properties]: On it.

File: reader-sdk-react-native-quickstart/android/app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -16,17 +16,16 @@ limitations under the License.
...
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
     <application
       android:name=".MainApplication"
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
       android:allowBackup="false"
-      android:theme="@style/SplashTheme">
```
[the remark this answers is not in the document]
Comment (jessdelacruzsantos) [reply reader-sdk-react-native-quickstart/android/app/src/main/AndroidManifest.xml]: It has not changed the behavior, when I reverted it broke the android build

File: reader-sdk-react-native-quickstart/android/app/src/main/res/values/styles.xml [hunk 7]
```diff
@@ -21,11 +21,6 @@ limitations under the License.
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
-        <item name="android:windowBackground">@drawable/window_background</item>
         <item name="android:alertDialogTheme">@style/Theme.AppCompat.Light.Dialog.Alert</item>
     </style>
-
-    <style name="SplashTheme" parent="AppTheme">
-        <item name="android:windowBackground">@drawable/launcher_screen</item>
```
[the remark this answers is not in the document]
Comment (jessdelacruzsantos) [reply reader-sdk-react-native-quickstart/android/app/src/main/res/values/styles.xml]: I don't see a difference from how the app ran before.

**Pull request merged**